### DISCUSSION TITLE
I have not received my student benefits, long after I have been approved which will affect my school assignment deadline

### DISCUSSION THREAD

**Discussion by noel-odero** [community/community#143880] [GitHub Education]
### Hi everyone,

Hello Github community, I was approved for github education benefits but I havent received my benefits yet. This is going to affect me in terms of school deadline for my assignments

**Comment by davevad93** [comment]
Hi @noel-odero ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @noel-odero was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by noel-odero** [comment]
Hello,
Thank you for your swift response. However, I have an assignment due this
weekend which I will not be able to do without the student pack. If
possible could you fasten the process for me? If I do not submit this
assignment on time I will have to repeat this module which is very costly.

Thank you for your understanding.

Sincerely,
@noel-odero

**Reply by davevad93** [reply]
Hey @noel-odero , sorry but I'm not from GitHub nor from the Education team so I can't expedite the process for you, and even so, GitHub no longer provides manual reviews, don't worry you'll get your benefits soon! 👍
